cfgov/v1/models/sublanding_page.py

```diff
@@ -72,13 +76,16 @@ class SublandingPage(CFGOVPage):
     objects = PageManager()
 
     def get_browsefilterable_posts(self, request, limit):
-        filter_pages = [p.specific for p in self.get_appropriate_descendants(request.site.hostname)
+        hostname = request.site.hostname
+        filter_pages = [p.specific for p in self.get_appropriate_descendants(hostname)
                         if 'FilterablePage' in p.specific_class.__name__
                         and 'archive' not in p.title.lower()]
         posts_tuple_list = []
         for page in filter_pages:
+            base_query = AbstractFilterPage.objects.live_shared(hostname).filter(CFGOVPage.objects.child_of_q(page))
+            logger.info('Filtering by parent {}'.format(page))
```

Comment:
Actually, I changed it back to `logger.info`, I tried updating the settings in `local.py` so that it would show debug, but then it was showing all this other stuff, and I don't feel like mucking with that in this PR.  Definitely something we should configure better later though.